SeventhHeavenUI/ViewModels/ControlMappingViewModel.cs

```diff
@@ -728,6 +728,24 @@ public bool IsCustomConfigOptionSelected
             }
         }
 
+
+        public static bool IsAutoRunChecked
+        {
+            get
+            {
+                return Convert.ToBoolean(Sys.FFNxConfig.Get("enable_auto_run"));
+            }
+            set
+            {
+                if (value)
+                {
+                    Sys.FFNxConfig.Set("enable_analogue_controls", "true");
+                }
+                Sys.FFNxConfig.Set("enable_auto_run", value.ToString());
```

Comment:
`value` should be a boolean, not a string. How does your `FFNx.toml` look like when the `.Save()` API is called?

Reply:
Save() asks for two strings. The FFNx.toml looks fine. Naturally, I tested it before making a PR.

Reply:
Looking at the definition, it looks like it might also accept booleans but Visual Studio was yelling at me to make it into a string so I went with that.